test/cli_stages.c

```diff
@@ -3,6 +3,8 @@
  * Copyright (c) 2015-2018 Intel, Inc. All rights reserved.
  * Copyright (c) 2015      Research Organization for Information Science
  *                         and Technology (RIST). All rights reserved.
+ * Copyright (c) 2018      Mellanox Technologies, Inc.
```

Comment:
Needs to be changed to 2015-2018.
Mellanox actually created all these files.
Please go over all other files in this directory and fix years.

Reply:
Done